docs/home/api-info/webhooks.mdx

```diff
@@ -20,37 +20,39 @@ To enable this integration approach with Metriport:
 1. Set this endpoint URL on the [Developers page](https://dash.metriport.com/developers) in the developer dashboard,
    or via the [Update Settings endpoint](/home/api-reference/settings/post-settings);
 1. This will generate a webhook key that should be used to authenticate requests on
-   your app's endpoint (webhook).
+   your app's endpoint (webhook) - see [authentication](#authentication) and 
+   [generating a new webhook key](#generating-a-new-webhook-key) below.
 
 General requirements for the endpoint:
 
 1. Public endpoint accessible from the internet;
 1. Does not do an HTTP redirect (redirects will not be followed);
 1. Accepts a `POST` HTTP request;
-1. Verifies requests by comparing the HTTP header `x-webhook-key` with the webhook key;
+1. Verifies requests using the HTTP header `x-metriport-signature` - see [authentication](#authentication) below;
 1. Accepts and responds to a [`ping` message](#the-ping-message);
+1. Be [idempotent](https://en.wikipedia.org/wiki/Idempotence) - it should accept being called more
+   than once with the same payload, with no changes to the end result.
 
-Additionally, depending on what Metriport APIs you're using, your endpoint will need to accept and process
-different messages when they become available - specifically:
+Additionally, your endpoint will need to accept and process different messages when they become available - specifically:
 
-1. If using the Devices API - accept and process [user data messages](#devices-api-messages).
-1. If using the Medical API - accept and process [patient data messages](#medical-api-messages).
+1. [Patient data messages](#medical-api-messages).
 
 ## Authentication
 
-When Metriport sends a webhook message, it includes an `x-metriport-signature` header. 
+When Metriport sends a webhook message, it includes an `x-metriport-signature` header.
 
 - `x-metriport-signature`: This is an [HMAC](https://en.wikipedia.org/wiki/HMAC) SHA-256 hash computed using your webhook key and the body of the webhook message.
```

Comment:
nit: formatting here is a bit weird - suggestion to combine the two:

When Metriport sends a webhook message, it includes an `x-metriport-signature` header - this is an [HMAC](https://en.wikipedia.org/wiki/HMAC) SHA-256 hash computed using your webhook key and the body of the webhook message.